django/db/backends/sqlite3/features.py

```diff
@@ -66,6 +57,25 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_default_keyword_in_insert = False
     supports_unlimited_charfield = True
 
+    @cached_property
+    def django_test_expected_failures(self):
+        failures = {
+            # The django_format_dtdelta() function doesn't properly handle mixed
+            # Date/DateTime fields and timedeltas.
+            "expressions.tests.FTimeDeltaTests.test_mixed_comparisons1",
+        }
+        if Database.sqlite_version_info < (3, 47, 0):
+            failures.update(
+                {
+                    # SQLite doesn't parse escaped double quotes in the JSON path
+                    # notation, so it cannot match keys that contains double quotes
+                    # (#35842).
+                    "model_fields.test_jsonfield.TestQuerying."
+                    "test_lookups_special_chars_double_quotes",
```

Comment:
There is no way it will be fixed in older versions of SQLite so I'd add it to the `django_test_skips` instead.